src/Sdk.php

```diff
@@ -52,6 +52,24 @@ function captureEvent(array $payload): ?string
     return Hub::getCurrent()->captureEvent($payload);
 }
 
+/**
+ * Logs the most recent error (obtained with {@link error_get_last}).
+ *
+ * @return null|string
+ */
+function captureLastError(): ?string
+{
+    $error = error_get_last();
+
+    if (null === $error || !isset($error['message'][0])) {
+        return null;
+    }
+
+    $exception = new \ErrorException(@$error['message'], 0, @$error['type'], @$error['file'], @$error['line']);
```

Comment:
Maybe we should pass default values instead of silencing errors?
```suggestion
    $exception = new \ErrorException(
        $error['message'] ?? null,
        0, 
        $error['type'] ?? null, // severity -- maybe E_ERROR?
        $error['file'] ?? null, // this by default risks being the file of this class, which would be wrong!
        $error['line'] ?? null // this too
    );
```